tests/target/multiple.rs

```diff
@@ -16,11 +16,7 @@ extern crate foo;
 extern crate foo;
 
 use std::cell::*;
-use std::{
-    self, any, ascii, borrow, borrow, borrow, borrow, borrow, borrow, borrow, borrow, borrow,
-    borrow, borrow, boxed, boxed, boxed, boxed, boxed, boxed, boxed, boxed, boxed, boxed, char,
-    char, char, char, char, char, char, char, char, char,
-};
+use std::{self, any, ascii, borrow, boxed, char};
```

Comment:
Wait, this is changing behavior without this flag.

However, this code is _broken_, so perhaps that's fine?